src/evergreen/api.py

```diff
@@ -164,10 +164,23 @@ def _call_api(
         :return: response from api server.
         """
         start_time = time()
+        LOGGER.debug(
```

Comment:
Do you think it would be beneficial to include how long it takes for the process to return?

Reply:
Yes and I think we are currently doing it in the `_log_api_call_time` function below.